packages/client/test/client.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('test utils', () => {
     ).to.be.equal('Message');
   });
   it('test Encrypt & Decrypt Long message', () => {
-    const message = "{'properties':[{'key':'name','type':0,'value':'Kim'},{'key':'age','type':0,'value':'17'},{'key':'address','type':0,'value':'Seoul'}],'sign':{'authType':0,'keyType':1,'value':'304402207282d176a100f0d5feb3faa160bd39843253ec98487cc3342905260ab592b85e02201fdf55464c12bbe4b93868a6e84da59e816e918d52cc599678605f61846426c6'}}\n";
+    const message = '{\'properties\':[{\'key\':\'name\',\'type\':0,\'value\':\'Kim\'},{\'key\':\'age\',\'type\':0,\'value\':\'17\'},{\'key\':\'address\',\'type\':0,\'value\':\'Seoul\'}],\'sign\':{\'authType\':0,\'keyType\':1,\'value\':\'304402207282d176a100f0d5feb3faa160bd39843253ec98487cc3342905260ab592b85e02201fdf55464c12bbe4b93868a6e84da59e816e918d52cc599678605f61846426c6\'}}\n';
```

Comment:
How about just reversing them?
```suggestion
    const message = '{"properties":[{"key":"name","type":0,"value":"Kim"}
    ...}\n';
```